openaerostruct/functionals/moment_coefficient.py

```diff
@@ -216,28 +264,62 @@ def compute_partials(self, inputs, partials):
                 partials["CM", name + "_b_pts"][0, :] = 0.0
                 partials["CM", name + "_b_pts"][1, :] *= 2.0
                 partials["CM", name + "_b_pts"][2, :] = 0.0
+                partials["M", name + "_sec_forces"][0, :] = 0.0
+                partials["M", name + "_sec_forces"][1, :] *= 2.0
+                partials["M", name + "_sec_forces"][2, :] = 0.0
+                partials["M", name + "_b_pts"][0, :] = 0.0
+                partials["M", name + "_b_pts"][1, :] *= 2.0
+                partials["M", name + "_b_pts"][2, :] = 0.0
                 dcda[0, :] = 0.0
                 dcda[1, :] *= 2.0
                 dcda[2, :] = 0.0
 
+            # Compute the derivative of CM wrt to the cg position which is negative dcda since diff = pts - cg times fact
+            # Accumlate the derivative over each surface as the total moment vector is sum over all surfaces.
             partials["CM", "cg"] -= dcda * fact
 
+            # Compute the derivative of M wrt to the cg position which is negative dcda since diff = pts - cg times fact
```

Comment:
I think `fact` shouldn't be there at the end of the comment? Might be a copy/paste error or maybe I'm wrong